Automattic/wp-calypso
# Akismet Pricing: Added plan constants for the new self serve plans.

## Proposed Changes

* Added new plan constants to the akismetts file

## Testing Instructions


 
* We are just adding the constants, so just a visual inspection of them should be fine. Additionally, we can ensure that Calypso loads fine using the live links for a sanity check. 

## Pre-merge Checklist
